core/src/test/java/io/fabric8/maven/core/service/openshift/PatchServiceTest.java

```diff
@@ -74,16 +75,18 @@ public void testResourcePatching() {
     }
 
     @Test
+    @Ignore
```

Comment:
Why?

Reply:
ah, sorry. I forgot to remove this. I remember I had fixed this test.